src/Microsoft.Health.Dicom.Core/Features/Common/DicomTagParser.cs

```diff
@@ -41,6 +42,8 @@ public bool TryParse(string dicomTagPath, out DicomTag[] dicomTags, bool support
 
         public string ParseFormattedTagPath(string dicomTagPath, bool supportMultiple = false)
         {
+            EnsureArg.IsNotNull(dicomTagPath, nameof(dicomTagPath));
```

Comment:
I know a lot of places we do `IsNotNullOrWhitespace` not sure if that's appropriate here or not.

Reply:
For now, given the volume of changes, I'll do the `null` check alone as if there was a `null` passed to this method today it would already throw an exception (albeit `NullReferenceException`).